usage_examples/multitask_openai/README.md

```diff
@@ -0,0 +1,42 @@
+# Performing multiple tasks in a single pipeline
+
+This example shows how you can perform multiple LLM-backed tasks within
+a single spaCy pipeline.
+
+We could create a new custom task that performs all objectives in a single
+LLM query, but in this example we'll only use built-in task templates to
+see how easy it is to compose them.
```

Comment:
Perhaps a word or two as to why we multiple tasks are better - maybe even a link to the [recent blog](https://explosion.ai/blog/against-llm-maximalism).

Reply:
Thanks. Done, let me know what you think